mscore/startupWizard.cpp

```diff
@@ -121,10 +121,23 @@ void StartupWizard::autoSelectShortcuts(QString keyboardLayout, QString os)
       layoutToShortcut->insert("Sebeolsik (세벌식)", ":/data/shortcuts.xml");
       layoutToShortcut->insert("OTHER", ":/data/shortcuts.xml");
     
+      QString os;
+#if defined(Q_OS_MAC)
+      os = "mac";
+#elif defined(Q_OS_WIN)
+      os = "win";
+#elif defined(Q_OS_LINUX)
+      os = "linux";
+#else
+      os = "other";
+#endif
+    
```

Comment:
Here too this isn't needed, just loading that shortcuts_MAC.xml if Q_OS_MAC is defined should do